guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
+In addition, you can bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign it to your host group or select it manually during provisioning.
+
+Using encrypted disks on hosts helps to proctect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
++
+* If you want to use Linux Unified Key Setup (LUKS) to encrypt and decrypt the disks of your host, add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
```

Comment:
Question just occurred to me. Is this step mandatory or optional? 

"If you want to use" sounds like this is an option. But the first sentence of this module sounds like LUKS encryption is always used. 

Please resolve the ambiguity.

Reply:
I have combined these two lines into one to clarify that a user has to select one option.